Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/divergences-data-mine-013-know-sure-thing-oscillator.json

```diff
@@ -0,0 +1,72 @@
+{
+    "topic": "Divergences Data Mine",
+    "pageNumber": 13,
+    "type": "Know Sure Thing Oscillator",
+    "definition": {
+        "text": "The Know Sure Thing Oscillator indicator was developed by Martin Pring in 1992 to make rate of change readings easier for traders to interpret.",
+        "updated": 1652685074675
+    },
+    "paragraphs": [
+        {
```

Comment:
Did this file happen on accident or is it planned to be a  new file?

Reply:
This is a new file.